codecparsers/nalReader_unittest.cpp

```diff
@@ -55,4 +55,31 @@ NALREADER_TEST(ReadBeyondBoundary)
     EXPECT_EQ(0, reader.readSe());
 }
 
+void checkBitread(NalReader& reader)
```

Comment:
It looks like this function is only relevant for checking a reader with a null/empty bitstream.  Perhaps rename it to checkBitreadEmpty or checkBitreadNull for clarity.